docs/release-notes.md

```diff
@@ -1,6 +1,37 @@
 # Release notes
 
-## Trino Gateway 10 (24 July 2024)
+## Trino Gateway 11 (4 Sep 2024)
+
+[JAR file gateway-ha-10-jar-with-dependencies.jar](https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/11/gateway-ha-11-jar-with-dependencies.jar),
+Docker container `trinodb/trino-gateway:11`
+
+* Add support to optionally deactivate adding `X-Forwarded-*` HTTP headers to
+  each request with the `addXForwardedHeaders` configuration in `routing`.
```

Comment:
```suggestion
  each request with the `addXForwardedHeaders: false` configuration in `routing`.
```
We are talking about how to *deactivate* the behavior.

Reply:
fair enough .. updated